src/post/Write/DeleteDraftModal.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Modal } from 'antd';
+import 'url-search-params-polyfill';
+import { injectIntl, FormattedMessage } from 'react-intl';
+import { deleteDraft } from './editorActions';
+import { notify } from '../../app/Notification/notificationActions';
+
+@injectIntl
+@connect(null, {
+  notify,
+  deleteDraft,
+})
+class DeleteDraftModal extends React.Component {
+  static propTypes = {
+    intl: PropTypes.shape().isRequired,
+    deleteDraft: PropTypes.func,
+    notify: PropTypes.func,
+    draftId: PropTypes.string,
+    onDelete: PropTypes.func,
+    onCancel: PropTypes.func,
+  };
+
+  static defaultProps = {
+    deleteDraft: () => {},
+    notify: () => {},
+    onDelete: () => {},
+    onCancel: () => {},
+    draftId: null,
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = {
```

Comment:
Can you set state outside of constructor? Constructor isn't necessary in this case.